libnatspec/natspecjs/package.json

```diff
@@ -1,24 +0,0 @@
-{
```

Comment:
Wait a second, I was not aware that this is an npm package. Perhaps we should check again whether this is in use somewhere.